lib/auth/webauthncli/fido2.go

```diff
@@ -711,7 +711,8 @@ func withRetries(callback deviceCallbackFunc) deviceCallbackFunc {
 				const msg = "" +
 					"The user verification function in your security key is blocked. " +
 					"This is likely due to too many failed authentication attempts. " +
-					"Consult your manufacturer documentation for how to unblock your security key."
+					"Device can be unblocked via teleport web client. " +
+					"You can also consult your manufacturer documentation for how to unblock your security key."
```

Comment:
Suggestion: end the message with the Web UI hint? I think the last sentence likely gets more weight.

```suggestion
					"Consult your manufacturer documentation for how to unblock your security key. " +
					"Alternatively, you may unblock your device by using it in the Web UI."

```